test/extended/testdata/build-secrets/s2i-binary-dir/.s2i/bin/run

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# Ensure none of the build config inject secrets still exist in the file system
+for s in $(ls /tmp/secret? secret?); do
```

Comment:
Since the expansion happens before the `for` is evaluated, you actually don't need the subshell or `ls`:
```bash
for secret in /tmp/secret? secret?; do
    ...
```